### PR TITLE
'get_cache' patch

### PR DESCRIPTION
**Pull request by brandoshmando**

### CODE REVIEW

**Comment by brandoshmando** [comment]
Needed for Django 1.9 https://docs.djangoproject.com/en/1.9/releases/1.9/#features-removed-in-1-9

**Comment by auvipy** [comment]
@brandoshmando  could u fix the merge conflict?

**Comment by mfenniak** [comment]
I happened to notice that this change seems to be in django-celery 3.1.17 on pypi, and v3.1.17 tag in the git repo (see: https://github.com/celery/django-celery/commits/v3.1.17), but not in the 3.1 branch, or the master branch.  Just thought you might want to know; seems like it was released but not merged?

**Comment by auvipy** [comment]
this change wasn't merged

**Comment by mfenniak** [comment]
Yes, but, it is present in the v3.1.17 tag and the 3.1.17 release on pypi.  Given that the PR hasn't been merged, that's what I thought was strange and should be brought to your attention. 🤔   It's odd to see something released but not merged.

**Comment by auvipy** [comment]
thanks will apply the change in next release

File: djcelery/backends/cache.py [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
+from django.core.cache import cache, caches
 
 from celery import current_app
 from celery.utils.timeutils import timedelta_seconds
 from celery.backends.base import KeyValueStoreBackend
 
 # CELERY_CACHE_BACKEND overrides the django-global(tm) backend settings.
 if current_app.conf.CELERY_CACHE_BACKEND:
-    cache = get_cache(current_app.conf.CELERY_CACHE_BACKEND)  # noqa
+    cache = caches(current_app.conf.CELERY_CACHE_BACKEND)  # noqa
```
Comment (vytisb) [inline djcelery/backends/cache.py]: This should use square brackets as `caches` is a dict-like object.

**Comment by auvipy** [comment]
superseded by another patch.

**Pull request closed without merging**